Update index.md

probichaux opened the pull request:
Added short section with a suggestion on how to politely ask for a call instead of using "hello" as a probe.

probichaux committed:
Update index.md

Added short section with a suggestion on how to politely ask for a call instead of using "hello" as a probe.

sbmueller commented:
Hi, thanks for the contribution. With the content being changed compared to the original nohello.com, I think the last paragraph needs to be adapted as well (e.g. not stating "This is a copy" but "This is inspired" or similiar). I will add some remarks in the code as well.

sbmueller commented on index.md:
```diff
@@ -41,6 +41,21 @@ communication. If the other party is away, and you leave before they come back,
+
+Instead of sending a call probe, or asking "hey, do you have time for a call?" 
+(which is better but not great), you can try something like this: 
+<pre>
```
Could you add a newline before `<pre>`?

probichaux replied:
Done

sbmueller commented on index.md:
```diff
@@ -41,6 +41,21 @@ communication. If the other party is away, and you leave before they come back,
+2010-07-19 12:32:12 you: Hi -- do you have X minutes for a quick call to talk about XYZ? [etc...]
+</pre>
+
+This does three things: it time bounds the request, it gives the recipient some context, and it helps you
```
Could you limit line length to 79 characters please?

probichaux replied:
Done

sbmueller commented on index.md:
```diff
@@ -41,6 +41,21 @@ communication. If the other party is away, and you leave before they come back,
 
+Some people like to probe with an initial "hello" to verify whether the presence 
+status shown is true, hoping they can determine whether the recipient is near a device,  
+then will immediately follow up with a call. Don't do that either. It's rude 
```
I would use less subjective wording here. Some people might interpret the behaviour as rude, however I would ask for a more neutral statement, like "If someone is able and willing to answer a chat message, it does not mean that they are also available for a voice or video call."

probichaux replied:
reworded per your suggestion.

probichaux committed:
Update index.md

Responded to comments in the original PR: fixed line length, added newline for pre tag, and edited to be less brusque.
